README.md

```diff
@@ -16,7 +16,7 @@ For detail about themes, please see [edx-docs about themes](https://edx.readthed
 
 Make sure you have MITx Online base url set in your settings. You can either add it in `env` files or simply add it in `edx-platform/lms/envs/private.py`. e.g
 
-`XPRO_BASE_URL="http://xpro.odl.local:8053/"`
+`MITX_ONLINE_BASE_URL="http://localhost:8013"`
```

Comment:
I was hoping to use `MARKETING_SITE_BASE_URL` for this, as in https://github.com/mitodl/ol-infrastructure/pull/305. I like it better because it's not project-specific and we could use it across projects. 
```suggestion
`MARKETING_SITE_BASE_URL="http://localhost:8013"`
```
If you've already used `MITX_ONLINE_BASE_URL` elsewhere, it's OK. Don't bother to change it.